apt/scripts/resolve-conflicts.py

```
@@ -0,0 +1,302 @@
+def parse_relation_names(value: str) -> set[str]:
+    """Extract package names from an apt relationship field."""
+    names: set[str] = set()
+    for part in value.split(","):
+        names.update(APT_RELATION_RE.findall(part.strip()))
+    return names
+
+
+def read_apt_fields(package: str) -> dict[str, list[str]]:
+    """Read fields from apt-cache show for one package."""
+    try:
+        result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)